lib/routes/hellogithub/volume.ts

```diff
@@ -10,53 +10,69 @@ const md = MarkdownIt({
     html: true,
 });
 import { load } from 'cheerio';
+import { fallback, queryToInteger } from '@/utils/readable-social';
+import cache from '@/utils/cache';
+import { config } from '@/config';
 
 art.defaults.imports.render = function (string) {
     return md.render(string);
 };
 
 export const route: Route = {
     path: ['/month', '/volume'],
-    name: 'Unknown',
-    maintainers: ['moke8', 'nczitzk'],
+    example: '/hellogithub/volume',
+    name: '月刊',
+    maintainers: ['moke8', 'nczitzk', 'CaoMeiYouRen'],
     handler,
 };
 
-async function handler() {
+async function handler(ctx) {
+    const limit: number = fallback(queryToInteger(ctx.req.query('limit')), 10);
```

Comment:
`fallback` is used to compare between 3 variables. You can use `||` or `??` for 2 variables.